### PR TITLE
feat: add drone-scp plugin.

### PR DESCRIPTION
**Pull request by appleboy**
Add [drone-scp](https://github.com/appleboy/drone-scp) plugin.

cc @bradrydzewski

### CODE REVIEW

File: content/appleboy/drone-scp/index.md [hunk 1]
```diff
@@ -0,0 +1,197 @@
+---
+date: 2017-01-06T00:00:00+00:00
+title: SCP
+author: appleboy
+tags: [ ssh, scp ]
+repo: appleboy/drone-scp
+logo: scp.svg
```
Comment (bradrydzewski) [inline content/appleboy/drone-scp/index.md]: let's use `term.svg` here (file already exists) and remove the scp logo. The scp logo in this pull request is for the SCP foundation which is not the same as the scp utility :)

File: content/appleboy/drone-scp/index.md [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+    image: appleboy/drone-scp
+    host: example.com
+    username: appleboy
+-   password: 12345678
+    port: 4430
+    target: /home/deploy/web
+    source: release.tar.gz
+```
+
+The above webhook Yaml attribute can be replaced with the `SCP_PASSWORD` secret environment variable. Please see the Drone documentation to learn more about secrets.
```
Comment (bradrydzewski) [inline content/appleboy/drone-scp/index.md]: we are making some changes to secrets in drone 0.6 and therefore have decided not to put example CLI commands in the documentation. Can you model this section after this example? https://github.com/drone/drone-plugin-index/blob/master/content/drone-plugins/drone-s3/index.md#secrets

File: content/appleboy/drone-scp/index.md (hunk 1 above)
Comment (appleboy) [inline content/appleboy/drone-scp/index.md]: Done!

File: content/appleboy/drone-scp/index.md (hunk 2 above)
Comment (appleboy) [inline content/appleboy/drone-scp/index.md]: Done!

**Comment by appleboy** [comment]
@bradrydzewski 

* replace logo
* update secret section.

Done!

**Comment by bradrydzewski** [comment]
awesome, thanks so much!

**Comment by appleboy** [comment]
@bradrydzewski Thanks.

**Pull request merged**